ujt/rpc_client.py

```diff
@@ -19,19 +19,28 @@
         StatusValue,
     )
 
-# Let's hardcode this for now... later can move into cfg file. Doesn't really fit in constants file
-# Not sure how to provide it as a cmd line argument from ujt module.
-channel = grpc.insecure_channel("localhost:50051")
-reporting_service_stub = server_pb2_grpc.ReportingServiceStub(channel)
+from . import config
+
+reporting_service_stub = None
+
+
+def connect():
+    global reporting_service_stub
+    channel = grpc.insecure_channel(config.REPORTING_SERVER_ADDRESS)
+    reporting_service_stub = server_pb2_grpc.ReportingServiceStub(channel)
```

Comment:
Same situation as the decorator, the previous code was run at load time and thus we can't use the configured value. This solution is really not ideal.

An alternative is to wrap these methods into a RpcClient class or something and have every module access a singleton instance. It seems more inconvenient for the callers, and also seems redundant to implement just to get the same functionality as global, so I decided to do this for now. Happy to change it if we can identify some clear drawbacks.